### PR TITLE
3669: Only display search result count without title.

### PR DESCRIPTION
**Pull request by rasben**
This commit removes the search result title,
and the count prefix around the result count.
We also change the logic for which count number
to use, based on the search result pagination.
See https://platform.dandigbib.org/issues/3669
for more info.

### CODE REVIEW

**Comment by rasben** [comment]
I think the circleci test timing out is unrelated to my actual code changes, but I dont have permissions to rebuild the test

File: modules/ting_search/plugins/content_types/search_result_count.inc [hunk 1]
```diff
@@ -22,25 +22,24 @@ function ting_search_result_count_content_type_render($subtype, $conf, $panel_ar
...
   if (isset($search_result)) {
-    $results = $search_result->getNumTotalObjects();
+    // OpenSearch only delivers the number of manifestions
+    // but the search result can also include materials.
+    // Because of this, we need to display different types
+    // of result counts based on the "more results" value,
+    // which basically checks if theres more than one page
+    // available for the search result pagination.
+    // @see https://platform.dandigbib.org/issues/979.
+    // @see https://platform.dandigbib.org/issues/3669
```
Comment (kasperg) [inline modules/ting_search/plugins/content_types/search_result_count.inc]: We generally do not refer to tickets in the comments. The link between ticket and commit should already be established through the commit message which includes the ticket id (I guess you could add multiple).

File: modules/ting_search/plugins/content_types/search_result_count.inc [hunk 2]
```diff
@@ -22,25 +22,24 @@ function ting_search_result_count_content_type_render($subtype, $conf, $panel_ar
 
   $search_result = ting_search_current_results();
   if (isset($search_result)) {
-    $results = $search_result->getNumTotalObjects();
+    // OpenSearch only delivers the number of manifestions
+    // but the search result can also include materials.
+    // Because of this, we need to display different types
+    // of result counts based on the "more results" value,
+    // which basically checks if theres more than one page
+    // available for the search result pagination.
```
Comment (kasperg) [inline modules/ting_search/plugins/content_types/search_result_count.inc]: Is there a reason why the comment is so cramped? Drupals Coding Standards requires a max line length of 80 chars. This seems to break at 60.

**Comment by rasben** [comment]
(This still needs review by another non-reload DDB developer)

File: modules/ting_search/plugins/content_types/search_result_count.inc [hunk 3]
```diff
@@ -22,25 +22,21 @@ function ting_search_result_count_content_type_render($subtype, $conf, $panel_ar
...
   if (isset($search_result)) {
-    $results = $search_result->getNumTotalObjects();
+    // OpenSearch only delivers the number of manifestions but the search 
+    // result can also include materials. Because of this, we need to display 
+    // different types of result counts based on the "more results" value,
+    // which basically checks if theres more than one page available for the 
+    // search result pagination.
+    $results = $search_result->hasMoreResults() ?
+      $search_result->getNumTotalObjects() :
+      $search_result->getNumTotalCollections();
```
Comment (JrgenGNielsen) [inline modules/ting_search/plugins/content_types/search_result_count.inc]: getNumTotalCollections() ændres til getNumCollections() jfr. https://platform.dandigbib.org/issues/3804

Comment (rasben) [reply modules/ting_search/plugins/content_types/search_result_count.inc]: Updated in d4a8b8b

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Pull request merged**